README.md

```diff
@@ -40,13 +40,21 @@ How to delete vpc and the reourses:
 	* NOTE: If the subnets had been existing, it will reuse the exsiting subnets in the zone
 
 	`$ ocmqe create subnets --region us-west-2 --zones a --vpc-id <vpc id>`
+    
 
 * Prepare proxy server
-	* NOTE: Only us-west-2 and ap-northeast-1 persisted the image.
-	* --ssh-file needs the openshift-qe.pem file path
-	* --ca-file just need a path to put the generated CA file
-
-	`$ ocmqe create proxy --region us-west-2 --zones a --vpc-id <vpc id> --ssh-file <openshift-qe.pem file path> --ca-file <ca-file path>`
+	* NOTE: Only us-west-2 and ap-northeast-1 persisted the image. 
```

Comment:
This line should be removed now. All regions support the proxy.

Reply:
Updated.